### ISSUE TITLE
ChoiceType array crash due to value name

### ISSUE THREAD

**Issue by danWithAD** [peter-murray/node-hue-api#170]
I've run into this simultaneously in local and remote running instance of node-hue-api. 

Error log is as follows, 

Failed to get a remote connection using existing tokens.
ApiError: Value 'allreadytoinstall' is not one of the allowed values [unknown,noupdates,transferring,anyreadytoinstall,allreadtoinstall,installing]

I've temporarily fixed this by adding the following to node_modules/node-hue-api/lib/types/ChoiceType.js
```

  _convertToType(val) {
    if (this.validValues.indexOf(val) > -1 || val === 'allreadytoinstall') {
      return val;
    } else {
      throw new ApiError(`Value '${val}' is not one of the allowed values [${this.validValues}]`);
    }
  }
```

**Comment by peter-murray** [comment]
It appears that hue has added another value to this choice type, https://github.com/peter-murray/node-hue-api/blob/80a6e4490c4955da9e8d934981a073f5efb65f76/lib/model/BridgeConfiguration.js#L31

I will modify the values and give you a pre release to test with later today.

**Comment by danWithAD** [comment]
Thank you so much!

**Comment by peter-murray** [comment]
This has been fixed in `5.0.0-alpha.2` (or at least it has in my local testing).

@danWithAD please verify when you get a chance

**Comment by danWithAD** [comment]
@peter-murray I can confirm that in both local and prod pulling the latest version from master repo works!

**Comment by dtsn** [comment]
@peter-murray did you patch 4.0.6? Just forced an update on 4.0.6 and I don't get the change. Looking at the tag in github it appears to be missing https://github.com/peter-murray/node-hue-api/blob/v4.0.6/lib/model/BridgeConfiguration.js#L36

**Comment by peter-murray** [comment]
@dtsn Sorry about that, looks like I failed to apply the actual commit for the fix and just cherry-picked two other commits that referenced it 🤦 

I have released `4.0.7` with the correct fix in it now, please try pulling that version from the npm registry.

**Comment by dtsn** [comment]
@peter-murray Works great! Thanks.
